lib/fs-loader.js

```diff
@@ -143,6 +143,23 @@ const load = async (dir, options, config) => {
   return Metaschema.create(schemas, config);
 };
 
+const asyncReadSchemas = async (path, systemParams, params) => {
```

Comment:
```suggestion
const readConfig = async (path, systemParams, params) => {
```
What do you think @lundibundi @belochub?